packages/@aws-cdk/aws-lambda/lib/lambda-version.ts

```diff
@@ -231,12 +231,12 @@ export class Version extends QualifiedFunctionBase implements IVersion {
   public metric(metricName: string, props: cloudwatch.MetricOptions = {}): cloudwatch.Metric {
     // Metrics on Aliases need the "bare" function name, and the alias' ARN, this differs from the base behavior.
     return super.metric(metricName, {
-      dimensions: {
+      dimensionsMap: {
```

Comment:
I'm not sure I understand why the name of this was changed.

Reply:
I changed this to fix a deprecation warning, because [MetricProps now prefers dimensionMap](https://docs.aws.amazon.com/cdk/api/v2/docs/aws-cdk-lib.aws_cloudwatch.MetricProps.html#dimensionsmap) instead of the older parameter.

Reply:
Ah, good catch.